tests/integration/imported-targets/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.10)
+
+project(import-targets
```

Comment:
Should this be "imported-targets" to match the folder name or should we rename the folder?

Reply:
Yes, it probably should be "imported-targets", I'll fix that.